### PR TITLE
[CF-361][FT] Fixed data collector in tests for rollbacks

### PR DESCRIPTION
**Pull request by plomakin**
Added password updating for migrated users on dst cloud.
To get key pairs we have to log in as each user and get list
of key pairs he owns. Since user password can not be migrated,
need to update it before log in.

### CODE REVIEW

File: cloudferry_devlab/cloudferry_devlab/tests/data_collector.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 import pkg_resources
 
 import yaml
+from keystoneclient import exceptions as ks_exceptions
```
Comment (SVilgelm) [inline cloudferry_devlab/cloudferry_devlab/tests/data_collector.py]: `keystone client` should be above `yaml`

File: cloudferry_devlab/cloudferry_devlab/tests/data_collector.py [hunk 2]
```diff
@@ -88,28 +89,51 @@ def nova_collector(self, destination, resources):
...
         if 'keypairs' in resources:
-            def get_user(name):
+            def get_user(usr):
+                def get_tenant_name(t_id):
+                    for t in cloud_tenants:
+                        if t_id == t.id:
+                            return t.name
+                    raise RuntimeError('Tenant with id "%s" not found' % t_id)
+
                 for _user in users:
```
Comment (SVilgelm) [inline cloudferry_devlab/cloudferry_devlab/tests/data_collector.py]: remove underscore from `_user`

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferry_devlab.tests.data_collector
cloudferry_devlab/cloudferry_devlab/tests/data_collector.py:18: [E0401(import-error), ] 
Unable to import 'keystoneclient'

**Comment by plomakin** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferry_devlab/cloudferry_devlab/tests/data_collector.py [hunk 3]
```diff
@@ -88,28 +89,51 @@ def nova_collector(self, destination, resources):
...
+            def get_user(usr):
+                for user in users:
+                    if user['name'] != usr.name:
+                        continue
+                    if user['tenant'] not in existing_tenants:
+                        try:
+                            user['tenant'] = existing_tenants[usr.tenantId]
+                        except KeyError:
+                            raise RuntimeError(
+                                "Tenant '%s' not found" % usr.tenantId)
```
Comment (roman-verchikov) [inline cloudferry_devlab/cloudferry_devlab/tests/data_collector.py]: This doesn't seem to be much cleaner than `KeyError`. Why is there no tenant? What to do with it?

**Comment by roman-verchikov** [comment]
-1, see inlines

**Comment by plomakin** [comment]
Closing this due to rollback job disabled.

**Pull request closed without merging**